src/nvim/search.h

```diff
@@ -3,6 +3,12 @@
 
 #include <stdbool.h>
 #include <stdint.h>
+#include "nvim/types.h" // for char_u
+#include "nvim/buffer_defs.h" // for buf_T, win_T
+#include "nvim/eval/typval.h" // for dict_T
+#include "nvim/normal.h" // for oparg_T
+#include "nvim/normal.h" // for cmdarg_T
+#include "nvim/os/time.h" // for Timestamp
```

Comment:
I would suggest to not add those comments at all. They are unmaintainable and serve no real purpose (nobody is actually checking whether they are true, not to mention whether they are complete).

Also `nvim/normal.h` is included twice, it should not be.